erdjs/src/smartcontracts/codec/binary.spec.ts

```diff
@@ -35,153 +36,186 @@ describe("test binary codec (basic)", () => {
 
         // Small int
 
-        check(BigInt(42), U8Type.One, [0x2A], [0x2A]);
-        check(BigInt(42), U16Type.One, [0x00, 0x2A], [0x2A]);
-        check(BigInt(42), U64Type.One, [0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x2A], [0x2A]);
-        check(BigInt(-10), I8Type.One, [0xF6], [0xF6]);
-        check(BigInt(-10), I16Type.One, [0xFF, 0xF6], [0xF6]);
+        check(BigInt(42), new U8Type(), [0x2A], [0x2A]);
```

Comment:
I think for each test this function should have a more descriptive name, at some point I thought the same function was used in all tests

Reply:
Fixed.